### ISSUE TITLE
Highlight on selected text isn't visible

### ISSUE THREAD

**Issue by christianstanfield** [james2doyle/atom-monokai-dark#7]

**Comment by Ustice** [comment]
Looks like the parent selector of atom-text-editor needs to use `::shadow`

This works in my in my `styles.less`

```` less
// Highlighted text
atom-text-editor::shadow {
  .selection .region {
    background-color: fade(rgb(248, 248, 242), 20%);
  }
}
````

**Comment by james2doyle** [comment]
@Ustice is that just for the hightlight? Or the whole element needs it?

**Comment by christianstanfield** [comment]
That worked for me too. Thanks!

**Comment by james2doyle** [comment]
Closed with #8
